transaction/core/src/tx_error.rs

```diff
@@ -15,6 +15,8 @@ pub enum NewTxError {
     Amount(AmountError),
     /// Memo: {0}
     Memo(NewMemoError),
+    /// Token Id not allowed at this block version
+    TokenIdNotAllowedAtBlockVersion,
```

Comment:
Suggestion: give a tell about which BlockVersion it was. It may be obvious to us which versions don't support TokenIds, but this'd be helpful to people new to developing against the chain.

```suggestion
    /// Token Id not allowed at block version: {0}
    TokenIdNotAllowedAtBlockVersion(BlockVersion),
```